src/clj/rems/api/catalogue.clj

```diff
@@ -15,6 +15,6 @@
 
     (GET "/" []
       :summary "Get the catalogue of items for the UI (does not include disabled)"
+      :roles #{:applicant}
       :return GetCatalogueResponse
-      (check-user)
```

Comment:
Why is check-user removed?

Reply:
refactor: use the declarative :roles support